rpc/tests/utils/client.rs

```diff
@@ -6,15 +9,16 @@ use jsonrpsee::core::client::ClientT;
 use jsonrpsee::core::Error;
 use jsonrpsee::http_client::HttpClient;
 use jsonrpsee::ws_client::WsClient;
-use once_cell::sync::Lazy;
-use std::env;
-use tokio::sync::Mutex;
-
-static LOCK: Lazy<Mutex<()>> = Lazy::new(|| Mutex::new(()));
+use tokio::sync::{Mutex, MutexGuard};
 
 const WS_URL: &str = "ws://localhost:26658";
 const HTTP_URL: &str = "http://localhost:26658";
 
+async fn lock() -> MutexGuard<'static, ()> {
```

Comment:
can we name it `write_lock`? I'd prefer it inlined in `submit_blob` but I guess you keep it separated in case in future we add another write api that needs that lock.